cmd/mayactl/app/command/volume_create_test.go

```diff
@@ -0,0 +1,54 @@
+package command
+
+import (
+	"fmt"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	utiltesting "k8s.io/client-go/util/testing"
+)
+
+func TestIsVolumeExist(t *testing.T) {
+	response := `{"items":[{"metadata":{"annotations":{"vsm.openebs.io/replica-status":"Pending,Running,Pending","openebs.io/jiva-replica-status":"Pending,Running,Pending","deployment.kubernetes.io/revision":"1","openebs.io/volume-monitor":"false","openebs.io/volume-type":"jiva","openebs.io/jiva-replica-count":"3","vsm.openebs.io/replica-ips":"nil,172.17.0.7,nil","openebs.io/jiva-replica-ips":"nil,172.17.0.7,nil","vsm.openebs.io/targetportals":"10.106.224.86:3260","vsm.openebs.io/cluster-ips":"10.106.224.86","openebs.io/storage-pool":"default","openebs.io/capacity":"5G","vsm.openebs.io/controller-ips":"172.17.0.3","openebs.io/jiva-controller-status":"Running","openebs.io/replica-container-status":"Running","openebs.io/jiva-iqn":"iqn.2016-09.com.openebs.jiva:test5","vsm.openebs.io/volume-size":"5G","openebs.io/controller-container-status":"Running","openebs.io/jiva-controller-cluster-ip":"10.106.224.86","vsm.openebs.io/iqn":"iqn.2016-09.com.openebs.jiva:test5","vsm.openebs.io/replica-count":"3","openebs.io/jiva-controller-ips":"172.17.0.3","vsm.openebs.io/controller-status":"Running","openebs.io/jiva-target-portal":"10.106.224.86:3260"},"creationTimestamp":null,"labels":{},"name":"test5"},"status":{"Message":"","Phase":"Running","Reason":""}}],"metadata":{}}`
+	tests := []*struct {
+		volname        string
+		fakeHandler    utiltesting.FakeHandler
+		addr           string
+		expectedOutput error
+	}{
+		{
+			volname: "test1",
+			fakeHandler: utiltesting.FakeHandler{
+				StatusCode:   200,
+				ResponseBody: string(response),
+				T:            t,
+			},
+			addr:           "MAPI_ADDR",
+			expectedOutput: nil,
+		},
+		{
+			volname: "test5",
+			fakeHandler: utiltesting.FakeHandler{
+				StatusCode:   200,
+				ResponseBody: string(response),
+				T:            t,
+			},
+			addr:           "MAPI_ADDR",
+			expectedOutput: fmt.Errorf("Error: Volume %v already exist ", "test5"),
+		},
+	}
+
+	for _, c := range tests {
+		server := httptest.NewServer(&c.fakeHandler)
```

Comment:
In subtests all the test cases run in parallel and results into faster test execution.can you make use of subtests instead of for loop ?